src/main/java/uk/gov/hmcts/reform/notificationservice/service/NotificationService.java

```diff
@@ -63,6 +63,8 @@ public void processPendingNotifications() {
             } catch (FeignException exception) {
                 postpone(notification, exception);
                 postponedCount++;
+            } catch (Exception e) {
+                postpone(notification, e);
```

Comment:
do we need to increment postponedCount ?

Reply:
fixed